README.md

```diff
@@ -17,11 +17,34 @@ npm install react-vapor
 > The project is at its early stages, some components can still have _lots_ of breaking changes between versions.
```

Comment:
While we're editing the README, can we remove this line? I feel like it's outdated.

Reply:
done 👍